src/config/dynamic-theme-fixes.config

```diff
@@ -5097,6 +5097,20 @@ INVERT
 
 ================================
 
+theinformation.com
+
+INVERT
+.logo
+
+CSS
+article.single .sticky-header button.button-share,
+#bookmark-and-share button.button-share {
+    background-color: #21498c;
```

Comment:
I assume this are the default values from the site itself?

The articles is after a paywall so I can't check for myself.

Reply:
These color values match other buttons on the site. You can confirm with the blue button on the [log-in page](https://www.theinformation.com/sessions/new).